tests/src/test/scala/org/apache/openwhisk/core/controller/test/RulesApiTests.scala

```diff
@@ -61,6 +63,11 @@ class RulesApiTests extends ControllerTestCommon with WhiskRulesApi {
   val activeStatus = s"""{"status":"${Status.ACTIVE}"}""".parseJson.asJsObject
   val inactiveStatus = s"""{"status":"${Status.INACTIVE}"}""".parseJson.asJsObject
   val parametersLimit = Parameters.sizeLimit
+  val dummyInstant = Instant.now()
+
+  def checkResponse(response: WhiskRuleResponse, expected: WhiskRuleResponse) =
```

Comment:
One small nit can be extracting this kind of methods into a trait.

Reply:
@style95 Thank you for review, I've created a common method to test the WhiskAction, WhiskActionMetaData, WhiskTrigger, and WhiskPackage. But I couldn't extract the method in the RulesApiTests because WhiskRuleResponse doesn't extend WhiskEntity.

Please review it again 😄 

https://github.com/apache/openwhisk/pull/4646/commits/26a446fbe9bdc97ea6cb8f3fc9e2be814c82e182#diff-a151ad71ab374ea12d3b9c780b59de30R88